web_src/less/_repository.less

```diff
@@ -377,10 +386,15 @@
             max-width: 250px;
           }
           width: 66%;
+
+          a:not(.ref-issue) {
+            color: var(--color-text-light-2);
+          }
```

Comment:
You can probably avoid this `:not` by using a `> a`  selector instead.

Reply:
Unfortunately it didn't work, as the default message link and issue links are siblings. Maybe I am missing something?

```html
<span class="truncate">
  <a href="..." class="default-link">Add support for http signatures (</a>
  <a href="..." class="ref-issue">#553</a>
  <a href="..." class="default-link">)</a>
</span>
```

But now that you mentioned it, I noticed that there other types of links that should be highlighted as well.
So I guess its safer to explicitly target `.default-link` instead of every link except `.ref-issue`.

![image](https://user-images.githubusercontent.com/7695608/179373621-2abd9ef0-2a1c-4fa6-be13-9323b7fb94aa.png)

Just pushed this change, will add new screenshots.